manifest.yml

```diff
@@ -0,0 +1,40 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"
+version: "202011.00"
+description: |-
+  "The AWS IoT Device SDK for Embedded C is a collection of C source
+  files that can be used in embedded applications to securely connect
+  to AWS IoT Core and interact with other AWS IoT services on AWS Cloud.\n
+  \n
+  See dependencies for included libraries."
+dependencies:
+  - name: "coreMQTT"
+    version: "v1.0.1"
+    repository:
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreMQTT"
+  - name: "coreJSON"
+    version: "v1.1.0"
+    repository:
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreJSON"
+  - name: "Device-Shadow-for-AWS-IoT-embedded-sdk"
+    version: "v1.0.1"
+    repository:
+      type: "git"
+      url: "https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk"
```

Comment:
Do the links contain capitalizations on purpose?

Reply:
Yes that is the name that was given to use orignally